pkg/apis/cloudcredential/v1/aws_types.go

```diff
@@ -49,3 +51,9 @@ type AWSProviderStatus struct {
 	// Policy is the name of the policy attached to the user in AWS.
 	Policy string `json:"policy"`
 }
+
+// AWSIAMCondition - map of condition types, with associated key - value mapping
+type IAMCondition map[string]IAMPolicyConditionKeyValue
```

Comment:
We want this to be consistent with other types, so either `IAMPolicyCondition` or `StatementEntryCondition` I would have said
```suggestion
// IAMPolicyCondition - map of condition types, with associated key - value mapping
type IAMPolicyCondition map[string]IAMPolicyConditionKeyValue
```